### PR TITLE
[ekf_localizer] remove input pose and twist topic

### PR DESCRIPTION
**Pull request by RyuYamamoto**
## Description
remove input pose and twist topic from ekf_localizer(only use covariance topic)

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: localization/pose_twist_fusion_filter/ekf_localizer/src/ekf_localizer.cpp [hunk 1]
```diff
@@ -51,15 +51,13 @@ EKFLocalizer::EKFLocalizer(const std::string & node_name, const rclcpp::NodeOpti
...
   pose_stddev_y_ = declare_parameter("pose_stddev_y", 0.05);
   pose_stddev_yaw_ = declare_parameter("pose_stddev_yaw", 0.035);
-  use_pose_with_covariance_ = declare_parameter("use_pose_with_covariance", false);
 
   /* twist measurement */
   twist_additional_delay_ = declare_parameter("twist_additional_delay", 0.0);
   twist_rate_ = declare_parameter("twist_rate", 10.0);  // used for covariance calculation
   twist_gate_dist_ = declare_parameter("twist_gate_dist", 10000.0);  // Mahalanobis limit
   twist_stddev_vx_ = declare_parameter("twist_stddev_vx", 0.2);
   twist_stddev_wz_ = declare_parameter("twist_stddev_wz", 0.03);
```
Comment (YamatoAndo) [inline localization/pose_twist_fusion_filter/ekf_localizer/src/ekf_localizer.cpp:60]: Let's remove these params too.

Comment (RyuYamamoto) [reply localization/pose_twist_fusion_filter/ekf_localizer/src/ekf_localizer.cpp:60]: @YamatoAndo OK. If user wants to adjust covariance of pose / twist, is it correct to imagine adjusting ndt_scan_matcher / vehicle_velocity_conveter side?

Comment (YamatoAndo) [reply localization/pose_twist_fusion_filter/ekf_localizer/src/ekf_localizer.cpp:60]: yes!!

Comment (RyuYamamoto) [reply localization/pose_twist_fusion_filter/ekf_localizer/src/ekf_localizer.cpp]: fixed f6a1cc9c4a5ffd91e8efb02a4328a2463c3a9dbc. I will fix launcher side.

**Review by YamatoAndo — APPROVED** [review]

**Comment by tkimura4** [comment]
I checked this PR with logging_simulator. It seemed to work well.

image

**Comment by RyuYamamoto** [comment]
@tkimura4 Sorry, now I was just checking this PR. I already checked x1 and xx1 proj and confirmed to work, so maybe no problem to merge this PR

**Pull request merged**